lib/chef-vault/item.rb

```diff
@@ -462,8 +462,8 @@ def add_client(client)
     # removes a client to the vault item keys
     # @param client_or_node [Chef::ApiClient, Chef::Node] the API client or node to remove
     # @return [void]
-    def delete_client_or_node(client_or_node)
-      client = load_actor(client_or_node.name, "clients")
+    def delete_client_or_node(name)
```

Comment:
The change to passing the name rather than the client object here was for memory benefits, yeah?

You should update the comments above the method, as it no longer takes an object.